sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -59,6 +71,163 @@ def get_service_statistics(self, **kwargs):
+        """Returns statistics for the given index, including a document count
+        and storage usage.
+
+        :param index_name: The name of the index to retrieve.
+        :type index_name: str
+        :return: Statistics for the given index, including a document count and storage usage.
+        :rtype: ~azure.search.documents.GetIndexStatisticsResult
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexes.get_statistics(index_name, **kwargs)
+        return result.as_dict()
+
+    @distributed_trace
+    def delete_index(self, index_name, **kwargs):
+        # type: (str, **Any) -> None
+        """Deletes a search index and all the documents it contains.
+
+        :param index_name: The name of the index to retrieve.
+        :type index_name: str
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        self._client.indexes.delete(index_name, **kwargs)
+
+    @distributed_trace
+    def create_index(self, index, **kwargs):
+        # type: (Index, **Any) -> Index
+        """
+
+        :param index: The index object.
+        :type index: ~azure.search.documents.Index
+        :return: The index created
+        :rtype: ~azure.search.documents.Index
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.indexes.create(index, **kwargs)
+        return result
+
+    @distributed_trace
+    def create_or_update_index(
+            self,
+            index_name,
+            index,
+            allow_index_downtime=None,
+            match_condition=MatchConditions.Unconditionally,
+            **kwargs
+    ):
+        # type: (str, Index, bool, MatchConditions, **Any) -> Index
+        """
+
+        :param index_name: The name of the index.
+        :type index_name: str
+        :param index: The index object.
+        :type index: ~azure.search.documents.Index
+        :param allow_index_downtime: Allows new analyzers, tokenizers, token filters, or char filters
+         to be added to an index by taking the index offline for at least a few seconds. This
+         temporarily causes indexing and query requests to fail. Performance and write availability of
+         the index can be impaired for several minutes after the index is updated, or longer for very
+         large indexes.
+        :type allow_index_downtime: bool
+        :param match_condition: The match condition to use upon the etag
+        :type match_condition: ~azure.core.MatchConditions
+        :return: The index created or updated
+        :rtype: ~azure.search.documents.Index
+        :raises: ~azure.core.exceptions.HttpResponseError,
+        ~azure.core.exceptions.ResourceNotFoundError
```

Comment:
Miss one space at least, check the output of sphinx I'm unsure if it's needed to have one `:raises:` per line